### PR TITLE
Added travis ci support for modern node versions

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

**Comment by RobLoach** [comment]
:+1:

**Comment by AdrieanKhisbe** [comment]
@amilajack could add the latest 7 and 8 too

**Comment by amilajack** [comment]
Done

**Comment by AdrieanKhisbe** [comment]
I was tooo optimistic about node 8 . :/

File: .travis.yml [hunk 1]
```diff
@@ -3,8 +3,10 @@ language: node_js
 node_js:
   - "0.10"
   - "0.12"
-  - "4.1"
-  - "5"
+  - 4
+  - 5
+  - 6
+  - 7
```
Comment (indexzero) [inline .travis.yml]: Please make `7` and `8` allowed failures for TravisCI. They will not officially be supported until `node@8` goes LTS in October.

**Comment by mhamann** [comment]
Will handle this via #280.

**Pull request closed without merging**